### PR TITLE
Add missing semi-colon

### PR DESCRIPTION
**Pull request by mbabker**
Per http://joomlacode.org/gf/project/joomla/tracker/?action=TrackerItemEdit&tracker_item_id=27843

### CODE REVIEW

File: libraries/cms/form/field/media.php [hunk 1]
```diff
@@ -65,7 +65,7 @@ protected function getInput()
 			$script[] = '	function jInsertFieldValue(value, id) {';
 			$script[] = '		var old_id = document.id(id).value;';
 			$script[] = '		if (old_id != id) {';
```
Comment (chdemko) [inline libraries/cms/form/field/media.php:67]: is the leading ';' useless  ?

Comment (mbabker) [inline libraries/cms/form/field/media.php:67]: Where at?

Comment (chdemko) [inline libraries/cms/form/field/media.php:67]: line 67

Comment (chdemko) [inline libraries/cms/form/field/media.php:67]: oops: my fault, I was misreading. Forgot that

**Pull request merged**